sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/service/schema/JsonSchemaManagerTest.java

```diff
@@ -0,0 +1,220 @@
+          objectMapper.readTree("{\"$schema\": \"https://json-schema.org/draft/2019-09/schema\"}");
+
+      jsonSchemaManager
+          .attachJsonSchema(namespace, collection, schema)
+          .subscribe()
+          .withSubscriber(UniAssertSubscriber.create());
+
+      verify(bridgeService).executeQuery(any(), any());
+      verifyNoMoreInteractions(bridgeService);
+    }
+
+    @Test
+    public void malformedSchema() throws JsonProcessingException {
+      String namespace = RandomStringUtils.randomAlphanumeric(16);
+      String collection = RandomStringUtils.randomAlphanumeric(16);
+
+      doAnswer(invocationOnMock -> null).when(bridgeService).executeQuery(any(), any());
+
+      JsonNode schema =
+          objectMapper.readTree(
+              "{\n"
+                  + "  \"$schema\": \"https://json-schema.org/draft/2019-09/schema\",\n"
+                  + "  \"type\": \"object\",\n"
+                  + "  \"properties\": {\n"
+                  + "    \"something\": { \"type\": \"strin\" }\n"
+                  + "  }\n"
+                  + "}");
+
+      jsonSchemaManager
+          .attachJsonSchema(namespace, collection, schema)
+          .subscribe()
+          .withSubscriber(UniAssertSubscriber.create())
+          .awaitFailure()
+          .assertFailedWith(ErrorCodeRuntimeException.class);
+
+      verifyNoMoreInteractions(bridgeService);
+    }
+  }
+
+  @Nested
+  class GetJsonSchema {
+
+    @Test
+    public void happyPath() throws JsonProcessingException {
+      table = Schema.CqlTable.newBuilder().putOptions("comment", "{\"schema\": {}}").build();
+
+      UniAssertSubscriber<JsonNode> result =
+          jsonSchemaManager
+              .getJsonSchema(Uni.createFrom().item(table))
+              .subscribe()
+              .withSubscriber(UniAssertSubscriber.create());
+
+      result.awaitItem().assertItem(objectMapper.readTree("{\"schema\": {}}")).assertCompleted();
+    }
+
+    @Test
+    public void noSchema() {
+      table = Schema.CqlTable.newBuilder().build();
+
+      UniAssertSubscriber<JsonNode> result =
+          jsonSchemaManager
+              .getJsonSchema(Uni.createFrom().item(table))
+              .subscribe()
+              .withSubscriber(UniAssertSubscriber.create());
+
+      result.awaitItem().assertItem(null).assertCompleted();
+    }
+
+    @Test
+    public void malformedJsonSchema() {
+      table = Schema.CqlTable.newBuilder().putOptions("comment", "lorem ipsum").build();
+
+      UniAssertSubscriber<JsonNode> result =
+          jsonSchemaManager
+              .getJsonSchema(Uni.createFrom().item(table))
+              .subscribe()
+              .withSubscriber(UniAssertSubscriber.create());
+
+      result.awaitItem().assertItem(null).assertCompleted();
+    }
+  }
+
+  @Nested
+  class ValidateJsonSchema {
+    @Test
+    public void happyPath() throws JsonProcessingException {
+      table = Schema.CqlTable.newBuilder().putOptions("comment", "{\"schema\": {}}").build();
```

Comment:
is empty schema the best for validation, why not taking the same one from the line `185`?